### PR TITLE
move some code out of clickhole

### PR DESCRIPTION
**Pull request by collin**
@MichaelButkovic starting to move this out of clickhole into bulbs.

### CODE REVIEW

File: tests/special_coverage/test_spec_cov_views.py [hunk 1]
```diff
@@ -45,6 +45,11 @@ def test_special_coverage_view(self):
         self.assertEqual(response.context['special_coverage'], sc)
         self.assertEqual(response.context['content_list'].count(), sc.get_content().count())
         self.assertEqual(response.context['content_list'][0].id, content.id)
+        self.assertEqual(response.context['recirc'], None)
```
Comment (collin) [inline tests/special_coverage/test_spec_cov_views.py]: This assertion is wrong.

The value here is an instance of: `djes.search.LazySearch`.

How strickt of validation should I go for here?

**Comment by collin** [comment]
See https://github.com/theonion/clickhole/pull/191

File: tests/special_coverage/test_spec_cov_views.py (hunk 1 above)
Comment (collin) [inline tests/special_coverage/test_spec_cov_views.py]: @MichaelButkovic I'm not sure exactly what I should be testing for in the result here.

Should I make some content and check for it?

I'm not quite sure how to set that test up.

Comment (MichaelButkovic) [inline tests/special_coverage/test_spec_cov_views.py]: Hm, yeah, I'd say add some content to it and then just assert that `recirc` doesn't have any unpublished data and that it doesn't have itself.

**Comment by collin** [comment]
@mparent61 or @benghaziboy can I get a 2nd here?

**Comment by MichaelButkovic** [comment]
:+1:

**Comment by mparent61** [comment]
:+1:

**Pull request merged**